tools/scripts/server/startServer.bat

```diff
@@ -1,30 +1,37 @@
 :: This script starts the local arma dev-server
-:: All meta-information are set via the setServerData.bat script that
+:: All meta-information are set via the setMetaData.bat script that
 :: is being called from this one
 @echo off
 
 :: set meta infos
-call %~dp0.\setMetaData.bat
+call "%~dp0.\setMetaData.bat"
+
+:: convert relative to absolute path of clib
+:: so armaserver can read it properly
+pushd "%~dp0..\..\..\PBOs\dev\@CLib"
+set "clib=%CD%"
+popd
+
+:: convert relative to absolute path of opt
+:: so armaserver can read it properly
+pushd "%~dp0..\..\..\PBOs\dev\@OPT"
+set "opt=%CD%"
+popd
 
 :: change directory into the Arma directory (in which the server-exe resides)
 cd /d "%armaDir%"
 
-set "clib=%modDir%\@CLib"
-set "opt=%modDir%\@OPT"
-
 :: Copy the server-config into the arma-dir as Arma can only read configs relative
 :: to the exe
-echo Trying to copy config file. This might take a while...
 :copyLoop
-	copy "%~dp0.\.\serverConfig.cfg" "%armaDir%" > NUL 2>&1
+	echo Trying to copy config file. This might take a while...
+	copy /Y "%~dp0.\serverConfig.cfg" "%armaDir%" > NUL 2>&1
 
 	if not [%errorlevel%] == [0]  (
-		:: sleep 100ms
-		pathping localhost -n -q 1 -p 100 >nul
+		:: sleep 1s
+		TIMEOUT /NOBREAK /T 1 > NUL
 		goto :copyLoop
 	)
 
 :: Start the server and minimize it once it's started
-start /min %exeName% -config=serverConfig.cfg -serverMod="%clib%;%opt%" -debugCallExtension
-
-echo Successfully started the server
\ No newline at end of file
+start /min %exeName% -config=serverConfig.cfg -serverMod="%clib%;%opt%;%additionalMods%" -debugCallExtension

```

Comment:
`%additionalMods%  aren't serverMods. They should be passed using the `-mod` parameter